specifications/di-bbs.json

```diff
@@ -1,7 +1,7 @@
 {
-  "name": "EdDSA Cryptosuite (v2022)",
-  "summary": "The Edwards Curve digital signature algorithm data integrity cryptosuite.",
-  "specification": "https://w3c.github.io/vc-di-eddsa/",
+  "name": "BBS Cryptosuite (v2023)",
```

Comment:
```suggestion
  "name": "The Data Integrity BBS Cryptosuites v1.0",
```

Reply:
I don't think we can apply this change, given the current draft title... but noting the difference between it and the other data integrity suites... I think we should strive for better alignment.

Reply:
cc @tplooker